docs/source/config-file.mdx

```diff
@@ -44,19 +44,50 @@ These fields are under the top-level `graphos` key and define your GraphOS graph
 | `apollo_registry_url`     | `URL`    |         | The URL to use for Apollo's registry                                                                            |
 | `apollo_uplink_endpoints` | `URL`    |         | List of uplink URL overrides. You can also provide this with the `APOLLO_UPLINK_ENDPOINTS` environment variable |
 
-### Forward Headers
+### Headers
+
+The `headers` option allows you to specify a list of hard-coded headers and values to include in all GraphQL requests.
+
+```yaml title="mcp.yaml"
+headers: { "apollo-client-name": "my-mcp-server" }
```

Comment:
I added this section because I was confused on the difference between `forward_headers` and `headers`. Wanted to confirm if this example code snippet is correct!